src/transformers/models/jukebox/configuration_jukebox.py

```diff
@@ -0,0 +1,667 @@
+            Number of blocks used in the `block_attn`. A sequence of length seq_len is factored as [blocks, seq_len //
+            blocks] in the `JukeboxAttention` layer.
+        encoder_depth (`int`, *optional*, defaults to 18):
+            Depth of the encoder model.
+        encoder_emb_dropout (`float`, *optional*, defaults to 0.0):
+            Embedding dropout used in the lyric encoder.
+        encoder_heads (`int`, *optional*, defaults to 4):
+            Number of heads in the lyric encoder
+        encoder_init_scale (`float`, *optional*, defaults to 0.1):
+            Initialisation scales for the lyric encoder modules.
+        encoder_loss_fraction (`list`, *optional*, defaults to [0.4, 0.0, 0.0]):
+            Multiplication factor used in front of the lyric encoder loss. Each value is for a particular level.
+        encoder_mlp_multiplier (`float`, *optional*, defaults to 1.0):
+            Multiplier coefficient used to define the hidden dimension of the MLP layers. 0.25 means that 0.25*width of
+            the model will be used.
+        encoder_n_vocab (`int`, *optional*, defaults to 79):
+            Defines the number of different lyric tokens that can be represented by the `inputs_ids` passed to the
+            `encoder`.
+        encoder_resid_dropout (`float`, *optional*, defaults to 0.0):
+            Residual dropout used in the attention pattern of the lyric encoder.
+        encoder_spread (`int`, *optional*, defaults to None):
+            Spread used in the `summary_spread_attention` pattern
+        encoder_width (`int`, *optional*, defaults to 128):
+            Width of the lyric encoder if `is_encoder_decoder=False` and `nb_relevant_lyric_tokens>0`
+        encoder_zero_out (`bool`, *optional*, defaults to False):
+            Whether or not to set to zeros the weights the convolutions in the lyric encoder.
+        init_scale (`float`, *optional*, defaults to 0.2):
+            Initialisation scales for the prior modules.
+        is_encoder_decoder (`bool`, *optional*, defaults to True):
+            Whether or not the prior is an encoder-decoder model. In case it is not, and `nb_relevant_lyric_tokens` is
+            greater than 0, the `encoder` args should be specified for the lyric encoding.
+        mask (`bool`, *optional*, defaults to False):
+            Whether or not to mask the previous positions in the attention.
+        max_duration (`int`, *optional*, defaults to 600):
+            _description_
+        max_nb_genres (`int`, *optional*, defaults to 1):
+            _description_
+        merged_decoder (`bool`, *optional*, defaults to True):
+            Whether or not the decoder and the encoder inputs are merged. This is used for the seperated
+            encoder-decoder architecture
+        metadata_conditioning (`bool`, *optional*, defaults to True):
+            _description_
+        metadata_dims (`tuple(int)`, *optional*, defaults to (604, 7898)):
```

Comment:
```suggestion
        metadata_dims (`tuple(int)`, *optional*, defaults to `(604, 7898)`):
```
Also tuples don't play well with JSON serialization, you should use lists.

Reply:
Got it, thanks for the warning will switch